backend/layers/persistence/persistence.py

```diff
@@ -892,6 +900,18 @@ def update_dataset_artifact(self, artifact_id: DatasetArtifactId, artifact_uri:
             artifact = session.query(DatasetArtifactTable).filter_by(id=artifact_id.id).one()
             artifact.uri = artifact_uri
 
+    def add_artifact_to_dataset_version(
```

Comment:
can we rename this to better differentiate it from [add_dataset_artifact](https://github.com/chanzuckerberg/single-cell-data-portal/blob/0d7c89d23b2097e5ca41076022af71665efa2520/backend/layers/persistence/persistence.py#L879)? and update description(s) accordingly?

maybe add_dataset_artifact_by_id? or something?